xml_converter/test_cases/can_fade/inputs/can_fade_is_correct.xml

```diff
@@ -1,8 +1,8 @@
 <OverlayData>
-<MarkerCategory  Name="iscorrect">
```

Comment:
This should probably be a value we expect to receive. Otherwise when we write the test that checks to make sure all category names exist and all poi types exist we might have to change all the tests. You can use a single name across all tests though such as `mycategory`.

Reply:
That's a good idea. Adding